values.sample.yaml

```diff
@@ -181,6 +181,10 @@ containerization:
     create: true
   datapacker:
     imageRepo: quay.io/tonicai/datapacker
+    serviceAccount:
+      create: false
+    environment:
+      configMaps: []
```

Comment:
Would it be wise to give a commented out example here, particularly for the serviceAccount section.
```yaml
    serviceAccount:
      create: true
      name: my-service-account
```
Otherwise the fact that `name` is an expected property of serviceAccount might go unnoticed.

Only important if we think customers will be setting this value

Reply:
I can add an example, I'll grab the manual IRSA configuration we're using.